benchmarks/models_baseline/bert_multi_gpu.yaml

```diff
@@ -0,0 +1,37 @@
+bert:
```

Comment:
do we need to have the baseline stored as files in the repo? Github lets us store artifacts which we can easily read. I was hoping we could do something like look at the last report or average of last n reports. This seems like it'll be annoying to maintain

Reply:
Ya I agree.. making these yaml files was painful... Let me look into github artifacts.

Reply:
This has been resolved with benchmark artifacts. We compute the average value based on last 7 reports